src/data/repositories/SurveyFormD2Repository.ts

```diff
@@ -2,19 +2,10 @@ import { D2Api } from "@eyeseetea/d2-api/2.36";
 import { D2TrackerEvent } from "@eyeseetea/d2-api/api/trackerEvents";
```

Comment:
General comment: yarn localize updates file: "i18n/es.po", it should be pushed to the PR.

Reply:
no updates to i18n/es.po file after running yarn localize.

Reply:
That's weird, I don't see the "es" translation for this message:

```
[arnau@ryzen9:amr-surveys]$ grep -r "This survey has other surveys associated with it" i18n/
i18n/en.pot:"This survey has other surveys associated with it.\n"
[arnau@ryzen9:amr-surveys]$ yarn localize &>/dev/null
[arnau@ryzen9:amr-surveys]$ grep -r "This survey has other surveys associated with it" i18n/
i18n/en.pot:"This survey has other surveys associated with it.\n"
i18n/es.po:"This survey has other surveys associated with it.\n"
```